notebook-reference/01-What are Notebooks.ipynb

```diff
@@ -0,0 +1,147 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## What are Notebooks?\n",
+        "Notebooks are an interactive tool for mixing code (C#, F#, Python, ...), their output/visualization, and documentation (Markdown) all in one spot.\n",
+        "\n",
+        "## What are they used for?\n",
+        "Notebooks are popular in Statistics, DataScience, and Machine Learning applications. Each of those fields use notebooks as a tool for trying things out, making changes, and seeing how it effects the output. The experimental and exploratory nature of Data Science and Machine Learning make Notebooks a great home but it's certainly not the only use for Notebooks!\n",
+        "\n",
+        "## How do I get started?\n",
+        "Hit play below ![Play Button](https://github.com/JakeRadMSFT/csharp-notebooks/raw/main/notebook-reference/images/play.png) to run a single cell or \"Run All\" ![Run All Button](https://github.com/JakeRadMSFT/csharp-notebooks/raw/main/notebook-reference/images/run%20all.png) above to run them all."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "dotnet_interactive": {
+          "language": "csharp"
+        }
+      },
+      "source": [
+        "Console.WriteLine(\"Hello Notebooks!\");"
+      ],
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": "Hello Notebooks!\r\n"
+          },
+          "execution_count": 1,
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "To add more cells you can use the buttons that show up when you hover above or below a cell ![Play Button](https://github.com/JakeRadMSFT/csharp-notebooks/raw/main/notebook-reference/images/hover%20buttons.png)  or keyboard shortcuts [Keyboard Shortcut Reference](https://github.com/JakeRadMSFT/csharp-notebooks/blob/main/notebook-reference/XX-VS%20Keyboard%20Shortcuts.ipynb)"
```

Comment:
Nit, I would make the sentence about keyboard shortcuts on a new line. The or is getting lost for me. 

![image](https://user-images.githubusercontent.com/9122518/169877005-98fe7a12-95bd-4672-9b71-fb4b8cc11cce.png)

I'd add a line break and then say "Keyboard shortcuts are also available."